From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 18 Nov 2017 15:35:03 +0100
Subject: [PATCH] {PRINTF} drivers/gpu/drm/amd/display: fix format strings

- "%%" to display "%"
- "%f" to format floats
- "%p" to format pointers
- missing format line
---
 drivers/gpu/drm/amd/display/dc/core/dc_stream.c        | 2 +-
 drivers/gpu/drm/amd/display/include/logger_interface.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_stream.c b/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
index 3732a1de9d6c..52315ae43b05 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_stream.c
@@ -326,7 +326,7 @@ void dc_stream_log(
 
 	dm_logger_write(dm_logger,
 			log_type,
-			"core_stream 0x%x: src: %d, %d, %d, %d; dst: %d, %d, %d, %d, colorSpace:%d\n",
+			"core_stream 0x%p: src: %d, %d, %d, %d; dst: %d, %d, %d, %d, colorSpace:%d\n",
 			stream,
 			stream->src.x,
 			stream->src.y,
diff --git a/drivers/gpu/drm/amd/display/include/logger_interface.h b/drivers/gpu/drm/amd/display/include/logger_interface.h
index f0e159131f9e..72ffedbd51d0 100644
--- a/drivers/gpu/drm/amd/display/include/logger_interface.h
+++ b/drivers/gpu/drm/amd/display/include/logger_interface.h
@@ -180,7 +180,7 @@ void context_clock_trace(
 	if (dc->debug.performance_trace) {\
 		dm_logger_write(dc->ctx->logger, \
 				LOG_PERF_TRACE, \
-				"%s duration: %d ticks\n", __func__,\
+				"%s duration: %lld ticks\n", __func__,\
 				perf_trc_end_stmp - perf_trc_start_stmp); \
 		if (perf_trc_start_log_msk != 1<<LOG_PERF_TRACE) {\
 			dc->ctx->logger->mask = perf_trc_start_log_msk;\
-- 
